### ISSUE TITLE
Simulated IMU

### ISSUE THREAD

**Issue by ruffsl** [fetchrobotics/fetch_gazebo#17]
I was testing out this package with gazebo and noticed that there was no imu messages being publish like for the real platform: http://docs.fetchrobotics.com/robot_hardware.html#imu

It'd be nice in general to add the IMUs in the gazebo models so I could test out projects like [cartographer](https://github.com/googlecartographer/cartographer_ros) that use senors fusion with IMUs in simulation. I suppose I could do a PR, but I'm not sure of the IMUs' origin wrt the robot's baselink and arm link.

**Comment by ruffsl** [comment]
Related issue pending on this enhancement:
https://github.com/googlecartographer/cartographer_fetch/issues/2#event-941048389

**Comment by moriarty** [comment]
Sorry @ruffsl this issue has been neglected a long time.

It's still very low priority but I've tagged @erelson and @ajzeller because they should be able to get the IMUs' origin wrt the robot's base link.

**Comment by erelson** [comment]
For future reference, internal ticket: HP-2127

If anyone wants to make a PR with dummy values, demonstrating what components of the URDF the they think the IMUs should be linked to, that may make this happen sooner :)
